updating node operation introduction (#1105)
onflow/flow: docs/content/node-operation/index.mdx

description: How to run your own Flow node
---

## Why run a node?

Running a node lets you participate in the network and contribute to the security and decentralization of the network.
A node also gives you direct access to the evolving state of the network.
Flow multirole architecture not it only makes it more scalable but also provides several node types that you as a node operator can pick and choose from.

> The different types of nodes are described [here](/node-operation/node-roles). As node operator, you can choose to run any of the different types of node that best fits your needs

Reviewer: needs period

description: How to run your own Flow node
---

## Why run a node?

Running a node lets you participate in the network and contribute to the security and decentralization of the network.
A node also gives you direct access to the evolving state of the network.
Flow multirole architecture makes it more scalable and provides several node types that you as a node operator can pick and choose from.

## Which node should you run?

The different types of nodes are described [here](/node-operation/node-roles). As node operator, you can choose to run any of the different types of node that best fits your needs.

### Observer node

> The observer node is one of the easiest node to spin up and is ideal for Dapps that need access to the latest block data available locally e.g a wallet application that needs to track the latest block ID and height.

Reviewer 2: ```suggestion
The observer node is one of the easiest nodes to spin up. Dapps interact with observer nodes when they need the latest block data locally eg. a wallet application that needs to track the latest block ID and height.
```

The observer node is not staked but still provides the same API as the access node.

> In addition to supporting dapps, an observer node can also be run by access node operators who want to scale their access nodes' endpoints geographically can spin up geographically dispersed observer nodes which can talk to their staked access nodes and to each other.
> To run an observer node, follow this [guide](/node-operation/observer-node).

Reviewer 2: ```suggestion
In addition to supporting dapps, an observer node can also be run by access node operators who want to scale their access nodes' endpoints. A user can spin up geographically dispersed observer nodes which can talk to their staked access nodes and to each other.
To run an observer node, follow this [guide](/node-operation/observer-node).
```
